packages/google_maps_flutter/lib/src/controller.dart

```diff
@@ -208,4 +208,25 @@ class GoogleMapController {
 
     return LatLngBounds(northeast: northeast, southwest: southwest);
   }
+
+  /// Return [ScreenCoordinate] of the [LatLng] in the current map view.
+  ///
+  /// A projection is used to translate between on screen location and geographic coordinates.
+  /// Screen location is in screen pixels (not display pixels) with respect to the top left corner
```

Comment:
I think this would be useful information on `ScreenInformation`'s DartDoc, too. I think it would be helpful to also say on the `x` and `y` members how the coordinate system works, eg is 0,0 top left or center?

Reply:
Updated this.